.devcontainer/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM eliteencoder/comfyui-base:latest
+ENV PATH="/miniconda3/bin:${PATH}"
+EXPOSE 8188
+EXPOSE 8889
+EXPOSE 3000
+EXPOSE 1024-65535/udp
+
+# Remove RunPod logo, replace with Livepeer AI
+RUN head -n $(($lines - 2)) ~/.bashrc > ~/.bashrc
+RUN echo -e "echo -e '\nFor detailed documentation and guides, please visit:\n\033[1;34mhttps://pipelines.livepeer.org/docs\033[0m and \033[1;34mhttps://mirror.xyz.livepeer.eth/\033[0m\n\n'" >> /root/.bashrc
```

Comment:
It's probably worth checking the licence of the runpod images. IIRC the NVIDIA ones are free to use as long as you maintain their headers on container startup. There might be sth similar for runpod (and worth checking if their licence does allow us to use it in our infra, as we are not using this only on runpod)

Reply:
Thanks for the tip! I was able to switch to `nvidia/cuda:12.2.2-cudnn8-devel-ubuntu22.04` (same as ai-worker) which is a smaller image, only had to add a few system packages like in ai-runner https://github.com/livepeer/ai-worker/blob/0b3edc633c22ba8507360c83955b5e3d14ed0f99/runner/docker/Dockerfile.live-base#L9-L13